README.md

```diff
@@ -1,86 +1,118 @@
 # AMD MIGraphX
 
-AMD's graph optimization engine.
+AMD MIGraphX is AMD's graph inference engine to accelerate model inference on AMD GPUs. AMD MIGraphX can be used by
+installing binaries directly or building from source code.
+
+Note that all the following instructions are based on that ROCm has been installed successfully. ROCm installation
+instructions are explained in the [ROCm installation
+guide](https://rocmdocs.amd.com/en/latest/Installation_Guide/Installation-Guide.html).
+
+## Installing from binaries
+With ROCm installed correctly, MIGraphX binaries can be installed with the following command:
+```
+sudo apt update && sudo apt install -y migraphx
+```
+then the head files and libs are located at `/opt/rocm/include` and `/opt/rocm/lib`, respectively, which can be
```

Comment:
First, it should be header and not head. Secondly, they are not located in /opt/rocm, unless the user adds a symlink. It should say that migraphx is installed under `/opt/roc,-<version>` where <version> is the rocm version.

Reply:
added.